tensilelite/README.md

```diff
@@ -0,0 +1,29 @@
+* TensileLite
+
+** How to Rebuild Object Codes Directly from Assembly
+
+During the tuning process, it is of interest to modify an assembly file/s and rebuild the corresponding object file/s and then relink the corresponding co file. Currently, we generate additional source files and a script to provide this workflow. 
+
+A new `Makefile` is added that manages rebuilding a co file during iterative development when tuning. One modifies an assembly file of interest, then runs `make` and make will detect what file/s changed and rebuild accordingly.
+
+Assumptions:
+
+- Each problem directory contains a library directory with one co file corresponding to one architecture
+
+Example:
+
+```
+cd hipBLASLt/tensilelite
+Tensile/bin/Tensile Tensile/Tests/gemm/fp16_use_e.yaml tensile-out
+# modify an assembly file in tensile-out/1_BenchmarkProblems/Cijk_Ailk_Bjlk_DB_UserArgs_00/00_Final/source/build_tmp/SOURCE/assembly
+make co TENSILE_OUT=tensile-out
+# re-run the client
+```
+
+The Makefile will set the target based on the name of the co file and sets a default wavefront flag but each of these can be customized as follows:
+
+```
+make co TENSILE_OUT=tensile-out ARCH="gfx942:xnack-" WAVEFRONTSIZE="-mwavefrontsize64"
+```
```

Comment:
For gfx942 example, it is better to remove WAVEFRONTSIZE="-mwavefrontsize64" since it is default setting
add another example for Wave32?
WAVE=32
example: make co TENSILE_OUT=tensile-out ARCH="gfx1201" WAVE=32

Reply:
Maybe not, sometimes these settings aren't automatically set correctly

Reply:
Which condition would make setting incorrect? We may list that in the README as well.

Reply:
@ellosel do you find out why sometimes the architecture cannot be detected correctly?